iquip/apps/scheduler.py

```diff
@@ -0,0 +1,293 @@
+        else:
+            self.nameLabel.setText("None")
+
+
+class ExperimentView(QWidget):
+    """Widget for displaying the information of the experiment."""
+
+    def __init__(self, info: ExperimentInfo, parent: Optional[QWidget] = None):
+        """Extended.
+        
+        Args:
+            info: The information of the experiment.
+        """
+        super().__init__(parent=parent)
+        # widgets
+        nameLabel = QLabel(info.name, self)
+        labels = (QLabel(f"{key}: {value}", self) for key, value in info.arginfo.items())
+        # layout
+        layout = QHBoxLayout(self)
+        layout.addWidget(nameLabel)
+        for label in labels:
+            layout.addWidget(label)
+
+
+class ExperimentModel(QAbstractListModel):
+    """Model for managing the data in the submitted experiment list.
+    
+    Attributes:
+        experimentQueue: The list of ExperimentInfo of queued experiments.
+    """
+
+    def __init__(self, parent: Optional[QWidget] = None):
+        """Extended."""
+        super().__init__(parent=parent)
+        self.experimentQueue: List[ExperimentInfo] = []
+
+    def rowCount(self, parent: Optional[QModelIndex] = QModelIndex()) -> int:  # pylint: disable=unused-argument
+        """Overridden."""
+        return len(self.experimentQueue)
+
+    def data(self,
+        index: QModelIndex,
+        role: Optional[Qt.ItemDataRole] = Qt.DisplayRole  # pylint: disable=unused-argument
+    ) -> ExperimentInfo:
+        """Overridden."""
+        return self.experimentQueue[index.row()]
+
+    def supportedDropActions(self) -> int:
+        """Overridden."""
+        return Qt.CopyAction | Qt.MoveAction
+
+    def mimeData(self, index: QModelIndex) -> QMimeData:
+        """Overridden.
+
+        Fetches the index of the selected element.
+        
+        Args:
+            index: The ModelIndex instance containing 
+              row, column, and etc. values of the selected element.
+        
+        Returns:
+            A QMimeData instance that holds str value of the index.
+        """
+        mime = super().mimeData(index)
+        mime.setText(str(index[0].row()))
+        return mime
+
+    def dropMimeData(self,  # pylint: disable=too-many-arguments
+        mimedata: QMimeData,
+        action: QAction,
+        row: int,
+        column: int,  # pylint: disable=unused-argument
+        parentIndex: QModelIndex  # pylint: disable=unused-argument
+    ) -> Literal[True]:
+        """Overridden.
+
+        Changes the priority of the experiments.
+        
+        Args:
+            mimedata: The QMimeData instance containing str value of the pre-selected index.
+            action: The QtAction instance classifying the action.
+              (for terminating the function when it is not dropped in the appropriate region)
+            row: The target row that is to be changed with the experiment in mimedata.
+            column: Not used.
+            parentIndex: Not used.
```

Comment:
The method `dropMimeData()` is just overridden and its argumetns already are in the `dropMimeData()` of the super class, so that I think the docstrings should be removed like `ExperimentDelegate.paint()`.